sync/src/orphan_block_pool.rs

```diff
@@ -12,13 +12,15 @@ const SHRINK_THRESHOLD: usize = 100;
 pub struct OrphanBlockPool {
     blocks: DashMap<ParentHash, HashMap<packed::Byte32, core::BlockView>>,
     parents: DashMap<packed::Byte32, ParentHash>,
```

Comment:
(Minor suggestion) Add a comment:

```suggestion
    // The map tells the parent hash when given the hash of a block in the pool.
    //
    // The block is in the orphan pool if and only if the block hash exists as a key in this map.
    parents: DashMap<packed::Byte32, ParentHash>,
```